.github/workflows/run-on-main-charts.yml

```diff
@@ -0,0 +1,18 @@
+# These set of workflows run on every push to the main branch
+name: Main build for Helm Charts
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
```

Comment:
Should this be done on tags and not all pushes to main?

Reply:
So the chart releaser will only publish when the chart versions have been bumped. The tag itself is performed by the chart-releaser action